cloth-diapers/cloth-diapers-basics.md

```diff
@@ -0,0 +1,296 @@
+very practical in that sense!
+
+<img width="492" alt="image" src="https://user-images.githubusercontent.com/4185328/170533205-f0cfd3c2-b9f6-49a2-8f9c-702ea9f271cf.png">
+
+
+### 2. Pocket Diapers
+We started using One Size diapers in the Spring and AIOs were a firm favourite
+because as I mentioned, the gave me very little work to put together.
+
+But then Winter came [insert your favourite Game of Thrones meme here].
+
+I struggled to get anything to dry, even with a dehumidifier on constantly.
+So we moved to pocket diapers for Winter.
+
+They take a LOT more time to put together but the shells air dry very quickly
+and if you're desperate you can throw the inserts in the tumble dryer too.
+
+We use _high quality_ natural material inserts.
+
+Our ***hemp inserts*** are triple layer hemp/cotton inserts with a _high_ percentage
+of hemp (55% hemp, 45% cotton): https://www.eco-tiny.com/product/absorvente-de-canhamo-e-algodao-organico-ecotiny
+
+Our ***bamboo inserts*** are 100% bamboo
+[Little Lamb bamboo doublers](https://littlelamb.com/products/bamboo-booster-doubler).
+
+Our infallible inserts combinations here are:
++ Morning diapers: 2 hemp inserts + 1 bamboo doubler folded in half (giving
+us 4 layers of bamboo booster) in the front of the diaper
++ Afternoon diapers: 1 hemp insert + 1 bamboo doubler folded in half (giving
+us 4 layers of bamboo booster) in the front of the diaper
+
+> We use our pockets throughout the Winter (with AIOs filling in the gaps)
+and whenever we are travelling for ease of washing and drying.
+
+
+### 3. Fitted diapers with covers (night time)
+For night time we use fitted diapers which are made entirely of absorbent material
+(in our case usually 100% bamboo or a bamboo/hemp mix), reinforced
+with hemp or bamboo inserts and covered with a PUL cover or wrap to make them
+leak proof.
+
+We wipe down the PUL cover in the mornings and it can be reused 3 times
+before needing to be washed (if baby is teething, the smells are often
+stronger so we wash as needed).
+
+We use these because:
++ We don't change diapers overnight because baby no longer :poop: and it
+would be worse to wake him for a diaper change
++ We need much more absorbency because of this - we could never get away
+with using a pocket overnight because we wouldn't be able to add enough
+absorbency (but a lot of people can, this depends on the baby)
++ We use hemp reinforcements because we don't have flooding (large amounts of
+liquid in a short space of time) so the slow
+absorption of hemp is fine for us
+  + If we needed to account for flooding I would test adding a cotton layer
+  between the baby and the hemp insert
+
+
+## Considerations
+
+### Morning vs Afternoons
+Babies will not have the same absorption needs all day or even all year round.
+Here, we require a lot more absorption in the mornings (until just after lunch)
+than in the afternoons.
+
+We have solved this pretty simply by splitting the diaper storage into a
```

Comment:
consider uploading a photo of your labeled shelves as it will add context to your organisation/system. 💭